packages/devtools_app/pubspec.yaml

```diff
@@ -188,3 +188,6 @@ dependency_overrides:
     path: ../devtools_shared
   devtools_test:
     path: ../devtools_test
+  vm_service: ^10.0.0
```

Comment:
Overrides should be removed before submitting (requires `vm_service` v.10.0.0 in flutter)